presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -912,6 +912,40 @@ procedure on the catalog's ``system`` schema::
     ``unregister_table`` only when using the Hive catalog. This is similar to
     the behavior listed above for the ``DROP TABLE`` command.
 
+Expire snapshots
+^^^^^^^^^^^^^^^^
+
+Each dml action in Iceberg produces a new snapshot while keeping the old data and metadata around for snapshot isolation and time travel. The expire_snapshots procedure can be used to remove older snapshots and their files which are no longer needed.
```

Comment:
```suggestion
Each DML (Data Manipulation Language) action in Iceberg produces a new snapshot while keeping the old data and metadata for snapshot isolation and time travel. Use `expire_snapshots` to remove older snapshots and their files.
```
Suggest capitalizing DML and adding the meaning of the acronym, other suggestions for active voice and conciseness.

Reply:
Thanks for the suggestion, fixed!